### PR TITLE
[CI/CD] Add FTR e2e test workflow for dashboards reporting

### PR DESCRIPTION
**Pull request by RyanL1997**
### Description
Add FTR e2e test workflow for dashboards reporting

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ps48 — APPROVED** [review]

**Comment by RyanL1997** [comment]
The log shows the FTR Cypress is up and running: https://github.com/opensearch-project/dashboards-reporting/actions/runs/7266094558/job/19797137965?pr=264#step:23:24

**Comment by RyanL1997** [comment]
Similar to the cypress workflow, this will be manual backported to `2.x`

File: .github/workflows/ftr-e2e-reporting-test.yml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+      - name: Get Cypress version
+        id: cypress_version
+        run: |
+          echo "::set-output name=cypress_version::$(cat ./package.json | jq '.dependencies.cypress' | tr -d '"')"
+        working-directory: opensearch-dashboards-functional-test
+
+      - name: Run Cypress tests
+        run: |
+          yarn cypress:run-without-security --browser chrome --spec 'cypress/integration/plugins/reports-dashboards/*.js'
```
Comment (mengweieric) [inline .github/workflows/ftr-e2e-reporting-test.yml:173]: Do we want to consider using Chromium with headless mode?

**Review by mengweieric — APPROVED** [review]

File: .github/workflows/ftr-e2e-reporting-test.yml (hunk 1 above)
Comment (RyanL1997) [reply .github/workflows/ftr-e2e-reporting-test.yml:173]: yes we can give a try.

Comment (RyanL1997) [reply .github/workflows/ftr-e2e-reporting-test.yml]: done

Comment (RyanL1997) [reply .github/workflows/ftr-e2e-reporting-test.yml]: After switch to chromium, the results remain the same.

**Pull request merged**